### DISCUSSION TITLE
Date identifier

### DISCUSSION THREAD

**Discussion by eoteroe** [JuliaLang/julia#48591] [General]
Hello, It would be great  adding an identifier for _dates_  and _datetimes_ and other Dates and time related formats in Julia.  Let's say: 

Date(2022,12,31) = D"2022-12-31"

**Comment by StefanKarpinski** [comment]
It's unlikely we'd use a single letter prefix for that, it would more likely be `date"2022-12-31"` at which point, what's the benefit over writing `Date(2022,12,31)`? It's the exact same number of characters. Is the desire to use the standard date format with compile time parsing?

**Reply by eoteroe** [reply]
I came from working from sql and i love how easy is to work with dates. Let’s say:  filtering, defining, etc. and when I see de Regex identifier r””, why not use the same principle.

**Reply by StefanKarpinski** [reply]
I kind of think that we should have used `re"..."` for regular expressions.

**Reply by eoteroe** [reply]
I just learnt other prefix: "raw". Many prefixes around. I guess it would be thinkable to expect a prefix for this important type: dt"2022-10-11 13:00:13". It'd be nice. Thank you.
